### PR TITLE
docs: PHPStan `Codeigniter` no type specified

### PR DESCRIPTION
**Pull request by ddevsr**
**Description**
Supersedes #7830

**Checklist:**
- [x] Securely signed commits
- [x] Component(s) with PHPDoc blocks, only if necessary or adds value
- [ ] Unit testing, with >80% coverage
- [ ] User guide updated
- [x] Conforms to style guide

### CODE REVIEW

File: system/CodeIgniter.php [hunk 1]
```diff
@@ -1092,6 +1116,9 @@ protected function sendResponse()
      * @param int $code
      *
      * @deprecated 4.4.0 No longer Used. Moved to index.php.
+     *
+     * @return void
```
Comment (paulbalandan) [inline system/CodeIgniter.php:1120]: just use never here. it is not a phpstan specific type

File: system/CodeIgniter.php [hunk 2]
```diff
@@ -1092,6 +1116,8 @@ protected function sendResponse()
      * @param int $code
      *
      * @deprecated 4.4.0 No longer Used. Moved to index.php.
+     *
+     * @return never
```
Comment (kenjis) [inline system/CodeIgniter.php:1120]: The `never` is correct type for this method, but this method is special one for testing purpose.

Why don' t you change to `@return void` to pass PHPStan checks?
After all, this method will be removed in the future.

File: phpstan-baseline.php [hunk 3]
```diff
@@ -3361,140 +1611,40 @@
...
-	'count' => 2,
-	'path' => __DIR__ . '/system/View/View.php',
-];
-$ignoreErrors[] = [
-	'message' => '#^Parameter \\#1 \\$node \\(PhpParser\\\\Node\\\\Stmt\\) of method Utils\\\\PHPStan\\\\CheckUseStatementsAfterLicenseRule\\:\\:processNode\\(\\) should be contravariant with parameter \\$node \\(PhpParser\\\\Node\\) of method PHPStan\\\\Rules\\\\Rule\\<PhpParser\\\\Node\\>\\:\\:processNode\\(\\)$#',
-	'count' => 1,
-	'path' => __DIR__ . '/utils/PHPStan/CheckUseStatementsAfterLicenseRule.php',
-];
 
 return ['parameters' => ['ignoreErrors' => $ignoreErrors]];
```
Comment (kenjis) [inline phpstan-baseline.php:1650]: You removed too many lines in `phpstan-baseline.php`.

Comment (ddevsr) [reply phpstan-baseline.php:1650]: Sorry, i missed merge conflict. Soon i'll make another PR fresh

Comment (ddevsr) [reply phpstan-baseline.php:1650]: Btw, setup job `PullRequests` still `read`

https://github.com/codeigniter4/CodeIgniter4/actions/runs/6195799415/job/16821194249?pr=7884

Comment (kenjis) [reply phpstan-baseline.php:1650]: Run `git rebase`.

**Pull request closed without merging**